tests/Support/SupportArrTest.php

```diff
@@ -682,5 +682,18 @@ public function testWrap()
         $this->assertEquals($array, Arr::wrap($array));
         $this->assertEquals([$object], Arr::wrap($object));
         $this->assertEquals([], Arr::wrap(null));
+        $this->assertEquals([null], Arr::wrap([null]));
+        $this->assertEquals([null, null], Arr::wrap([null, null]));
+        $this->assertEquals([''], Arr::wrap(''));
+        $this->assertEquals([''], Arr::wrap(['']));
+        $this->assertEquals([false], Arr::wrap(false));
+        $this->assertEquals([false], Arr::wrap([false]));
+        $this->assertEquals([0], Arr::wrap(0));
+
+        $obj = new stdClass;
+        $obj->value = 'a';
+        $obj = unserialize(serialize($obj));
```

Comment:
Why is this line needed?

Reply:
Following https://github.com/laravel/framework/pull/26966 failing PR, we decided to add tests preventing that mistake.
Also check : https://secure.php.net/manual/en/function.is-object.php